lib/commanded/event/handler.ex

```diff
@@ -639,6 +743,11 @@ defmodule Commanded.Event.Handler do
             inspect(module) <> " specifies invalid options: " <> inspect(Keyword.keys(invalid))
     end
 
+    if Keyword.has_key?(config, :concurrency) and Keyword.has_key?(config, :batch_size) do
+      raise ArgumentError,
+            "both `:concurrency` and `:batch_size` are specified, this is not yet supported. Please choose one or the other."
+    end
```

Comment:
Any reason why both of these options cannot be supported by Commanded? The eventstore will support batch size for multiple consumers and will attempt to distribute events appropriately.

Reply:
We don't use concurrency at the moment, and this is a bit of a scratch-your-own-itch PR, of course (as usual). Thinking through single-threaded batching was already complicated enough and we really want to dogfood code we submit, so we decided to make it a "not supported" thing for now. The "yet" is there to entice someone who does want/need it to fix that gap :)